dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -119,8 +119,10 @@
 
     protected static final JavaInfo javaInfo = JavaInfo.forCurrentVM();
 
-    protected static final boolean GLOBAL_JAVA2SECURITY = PrivHelper.getBoolean("global.java2.sec");
-    protected static final boolean GLOBAL_DEBUG_JAVA2SECURITY = PrivHelper.getBoolean("global.debug.java2.sec");
+    protected static final boolean GLOBAL_JAVA2SECURITY = FATRunner.FAT_TEST_LOCALRUN //
+                    ? Boolean.parseBoolean(PrivHelper.getProperty("global.java2.sec", "true")) //
+                    : Boolean.parseBoolean(PrivHelper.getProperty("global.java2.sec", "false"));
+    protected static final boolean GLOBAL_DEBUG_JAVA2SECURITY = Boolean.getBoolean("global.debug.java2.sec");
```

Comment:
Deliberate change from `PrivHelper.getBoolean` to `Boolean.getBoolean` ?

Reply:
Is J2S already on at this point? I suspect it isn't since that is what the properties are doing ... so actually, the other calls don't need to use PrivHelper?

Reply:
No this wasn't an intentional change -- I'll change it back.  Also, for most cases I don't think j2sec is on at this point so Tom is righ that PrivHelper probably isn't needed, but there may be some environments I'm not aware of where j2sec is enabled in a different way than the global.java2.sec property.